clang/lib/3C/3C.cpp

```diff
@@ -784,7 +785,9 @@ bool _3CInterface::invalidateWildReasonGlobally(ConstraintKey PtrKey) {
 
   // Delete ALL the constraints that have the same given reason.
   VarAtom *VA = CS.getOrCreateVar(PtrKey, "q", VarAtom::V_Other);
-  Geq NewE(VA, CS.getWild());
+  // TODO: there was previously no reason here. This code may have relied on
+  // lack of reasons elsewhere.
+  Geq NewE(VA, CS.getWild(), ReasonLoc());
```

Comment:
`invalidateWildReasonGlobally` is dead with the clangd3c removal; you can just delete it rather than adding a TODO if you prefer.  `makeSinglePtrNonWild` is also dead.